context/app/static/js/components/Detail/SummaryData/SummaryData.jsx

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { capitalizeString } from 'helpers/functions';
 import { Flex, FlexRight, FlexCenterAlign, JsonButton, StyledFileIcon } from './style';
 import SummaryItem from '../SummaryItem';
 import StatusIcon from '../StatusIcon';
 
 function SummaryData(props) {
-  const { entity_type, uuid, status, children } = props;
+  const { entity_type, uuid, status, data_access_level, children } = props;
 
   return (
     <Flex>
       {children && <FlexCenterAlign>{children}</FlexCenterAlign>}
       <FlexRight>
-        {entity_type === 'Dataset' && status && status.length > 0 && (
+        {entity_type === 'Dataset' && (
           <FlexCenterAlign>
             <StatusIcon status={status} />
             <SummaryItem>{status}</SummaryItem>
+            <SummaryItem>{`${capitalizeString(data_access_level)} Access`}</SummaryItem>
```

Comment:
After the next reindex, there will be an uppercased version of this, but we'll simplify this later.